src/components/Filters/FilterChecklist.vue

```diff
@@ -173,8 +185,7 @@ export default {
 
 <style lang="scss" scoped>
 .filters {
-  border-bottom: 2px solid rgb(245, 245, 245);
-  padding: 1.5rem 1rem 1.5rem 1.5rem;
+  padding: 1rem 1.5rem 1rem 52px;
```

Comment:
```suggestion
  padding: 1rem 1.5rem 1rem 3.25rem;
```

Best not to mix units.